queries.sql

```diff
@@ -8,3 +8,68 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+SELECT * FROM animals;
+
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+
+BEGIN;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT Point1;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO Point1;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+COMMIT;
+
+SELECT * FROM animals;
+
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
+
+SELECT species, MIN(weight_kg) FROM animals GROUP BY species;
+
+SELECT species, MAX(weight_kg) FROM animals GROUP BY species;
```

Comment:
- [ ] Well! You are currently showing the minimum and maximum animal weight by species in two separate queries. However, as per the project requirement, "What is the minimum and maximum weight of each type of animal?" it should be done in one query. Please consider updating your query to meet this requirement and add an updated screenshot in the PR description. Consolidating these results into a single query will align your code more closely with the project's specifications. Great effort! 👍📊🐾